File: charArrayAndString/numberOfWordsAndSentences.cpp
#include<iostream>
#include<cstdlib>
using namespace std;
int main(){
	char sent[50] = "Bugun sen cok gencsin yavrum";
	int wordC = 0, sentC = 1, i = 0;
	while(sent[i] != '\0'){
		if(isspace(sent[i]) != 0){
			sentC++;
		}
		wordC++;
		i++;
	}
	cout << "Word count = " << wordC << endl;
	cout << "Sentences count = " << sentC << endl;
 	return 0;
}

File: charArrayAndString/numberOfWordsAndSentences.java
package springIntro;

import java.util.*;

public class numberOfWordsAndSentences {

	public static void main(String[] args) {
		String sent = "Bugun sen cok gencsin yavrum";
		int wordsC = 0, sentC = 1;
		for(int i = 0; i < sent.length(); i++) {
			if(sent.charAt(i) == ' ') {
				sentC++;
			}
			wordsC++;
		}
		System.out.printf("Word count = %d\nSentences count = %d", wordsC, sentC);
	}

}
